src/index.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
Hi Clinton! Well done on this task, I think we should be using react-router-dom and not react-dom. Also, I think you should be declaring the router in the entry file- index.js and not in the container. overall you did a good job.

Reply:
Thanks for noticing that. That is not supposed to be part of this PR. Have fixed that. Thanks